Tools/Taxonomy/Cwe/CweTaxonomyGenerator.cs

```diff
@@ -139,10 +139,10 @@ private Run ConvertToSarif(List<CweCsvRecord> records, string version)
             IList<ToolComponent> taxonomies = new List<ToolComponent>();
             ToolComponent cweTaxonomy = new ToolComponent
             {
-                Name = Constants.CWE.Name,
-                Guid = Constants.CWE.Guid,
+                Name = Constants.CWE_Comprehensive_V43.Name,
+                Guid = Constants.CWE_Comprehensive_V43.Guid,
                 Version = version,
-                ReleaseDateUtc = Constants.CWE.ReleaseDate,
+                ReleaseDateUtc = Constants.CWE_Comprehensive_V43.ReleaseDate,
                 InformationUri = new Uri("https://cwe.mitre.org/data/published/cwe_v4.4.pdf"),
                 DownloadUri = new Uri("https://cwe.mitre.org/data/xml/cwec_v4.4.xml.zip"),
```

Comment:
>DownloadUri [](http://example.com/codeflow?start=16&length=11)

this does not look right.
The version is 43, but its point to 44

Reply:
forgot to change the url, fixed